### PR TITLE
fix(packages/agent) fix v0.x auto options

### PR DESCRIPTION
**Pull request by 9renpoto**
# TL;DR

Closes https://github.com/userdive/agent.js/issues/431

Connect to https://github.com/userdive/agent.js/pull/425/files#diff-363bc5c09c492691317051835a9df4b0R28

### CODE REVIEW

File: packages/agent/src/global.ts [hunk 1]
```diff
@@ -73,18 +75,18 @@ export default function (
...
+          : warning(`execute timeout: ${cmd}`, lazyStack)
       }
       const [next, ...queue] = obj2array(q)
-      applyQueue(next, queue)
+      applyQueue(next, queue, lazyStack)
     }, lazyStack[cmd] * lazyStack[cmd] * 100)
   }
 
   if (w[name] && w[name].q) {
     const [next, ...queue] = obj2array(w[name].q)
```
Comment (9renpoto) [inline packages/agent/src/global.ts]: oops ~ remove queue cache

File: packages/agent/src/global.ts [hunk 2]
```diff
@@ -73,18 +75,18 @@ export default function (
...
-    setTimeout(() => applyQueue(next, queue), 0)
+    setTimeout(() => applyQueue(next, queue, {}), 0)
```
Comment (9renpoto) [inline packages/agent/src/global.ts]: remove async call

File: packages/agent/src/global.ts [hunk 3]
```diff
@@ -48,43 +48,38 @@ const execute = (Agent: AgentClass, agents: { [key: string]: any }) => (
...
   agents: { [key: string]: any },
-  name: string,
-  w: any
+  name: string
 ) {
-  const applyQueue = (argsObject: any[], q: Arguments[]) => {
-    if (!argsObject || !argsObject[0]) {
-      return
-    }
+  const applyQueue = (delay: number) => {
```
Comment (dkimura) [inline packages/agent/src/global.ts]: async await使いたい件はここかな？

File: packages/agent/src/global.ts [hunk 4]
```diff
@@ -25,7 +25,7 @@ const execute = (Agent: AgentClass, agents: { [key: string]: any }) => (
     if (typeof args[1] === 'object') {
       // _ud('create', 'id', { auto: true, name: 'myTracker' })
       args[2] = args[1]
-      args[1] = args[1].auto || args[1].cookieName
+      args[1] = args[1].auto && 'auto'
```
Comment (dkimura) [inline packages/agent/src/global.ts]: https://uncovertruth.slack.com/archives/C02NHQJ61/p1522376518000151

File: packages/agent/src/global.ts (hunk 3 above)
Comment (9renpoto) [reply packages/agent/src/global.ts]: pluginをdynamic importでawaitできればこんな雑なdelayを書かなくてもよかった...

**Pull request merged**